DataConnectors/Purview/AzurePurview.json

```diff
@@ -0,0 +1,73 @@
+{
+    "id": "MicrosoftAzurePurview",
+    "title": "Azure Purview",
+    "publisher": "Microsoft",
+    "descriptionMarkdown": "Connect to Azure Purview. Azure Purview is a unified data governance service that helps you manage and govern your on-premises, multicloud, and software-as-a-service (SaaS) data. It creates a holistic, up-to-date map of your data landscape with automated data discovery, sensitive data classification, and end-to-end data lineage that empowers you to find valuable and trustworthy data.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "PurviewDataSensitivityLogs",
+            "baseQuery": "PurviewDataSensitivityLogs"
+        },
+        {
+            "metricName": "Total data received",
+            "legend": "PurviewScanStatusLogs",
+            "baseQuery": "PurviewScanStatusLogs"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "All scan logs",
+            "query": "PurviewScanStatusLogs\n | sort by TimeGenerated"
+        },
+        {
+            "description" : "View files that contain a specific classification (example shows Social Security Number)",
+            "query": "PurviewDataSensitivityLogs\n | where Classification contains \"Social Security Number\""
```

Comment:
Sample query is giving error while executing for **PurviewDataSensitivityLogs** as
**Failed to resolve table or column expression named 'PurviewDataSensitivityLogs'**

Reply:
The PurviewDataSensitivityLogs are still being onboarded, will update once the logs are ready and the sample query can be tested.

Reply:
Thanks for the Update Lisa.

Do you want us to wait for this PR to be merged until **PurviewDataSensitivityLogs** is onboarded?

Reply:
Yes, we can wait until the logs are onboarded. Hopefully, it will happen soon.

Reply:
We got the PurviewDataSensitivityLogs onboarded in canary and I was able to successfully test the sample queries. I also removed the PurviewScanStatusLogs as we will only have the PurviewDataSensitivityLogs for the private preview. We are ready to move forward with merging the PR.

Reply:
@libarson , Can you please tell us how to test these queries and in which LA workspace is this PurvviewDataSensitivityLogs onboarded? Still do not see that.
![image](https://user-images.githubusercontent.com/73506651/123558741-bd660c00-d74c-11eb-9157-e5c000cafe7d.png)

Reply:
@Saasri Can you help out in giving guidance on where the PurviewDataSensitivityLogs are onboarded and how to test them?

Reply:
![image](https://user-images.githubusercontent.com/30126939/123677263-bea14280-d7f9-11eb-88ef-4fcb64d41dde.png)